filament/CMakeLists.txt

```diff
@@ -152,17 +154,6 @@ set(MATERIAL_SRCS
         src/materials/skyboxRGBM.mat
 )
 
-# The noop driver is only useful for ensuring we don't have certain build issues.
-# Remove it from release builds, since it uses some space needlessly.
-if (CMAKE_BUILD_TYPE MATCHES Debug)
-    list(APPEND SRCS src/driver/noop/NoopDriver.cpp)
```

Comment:
Why are you putting the noop driver back in the release builds?

Reply:
Nevermind, you mention it in the PR description. Could we add `#ifndef NDEBUG` inside those files then so we don't take up extra space in Andriod release builds?

Reply:
Yup, that works too, will do